app/src/main/java/com/jetbrains/handson/mpp/mobile/CustomTableCell.kt

```diff
@@ -0,0 +1,20 @@
+package com.jetbrains.handson.mpp.mobile
+
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.jetbrains.handson.mpp.mobile.dataObjects.frontendDataObjects.JourneyTableDataElem
+
+class CustomTableCell(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    fun fillInfo(journey: JourneyTableDataElem) {
+        with(itemView) {
+            findViewById<TextView>(R.id.carrier).text = journey.trainOperator
+            findViewById<TextView>(R.id.depStation).text = journey.startStation.shortName
+            findViewById<TextView>(R.id.depTime).text = journey.getStartTime()
```

Comment:
Probably clear in this case but I wouldn't worry about name length too much, you could probably call this departureStation/arrivalTime/etc